combat-trainer.lic

```diff
@@ -65,6 +70,27 @@ class SetupProcess
 
   private
 
+  def blow_warhorn()
+    return unless @warhorn
+    return if Time.now - @last_warhorn < 300
+
+    if DRCI.wearing?(@warhorn)
```

Comment:
Would this logic be better done once? Maybe in the init?

Reply:
done!